tumblr-photo-video-ripper.py

```diff
@@ -214,6 +218,9 @@ def _download_media(self, site, medium_type, start):
             except UnicodeDecodeError:
                 print("Cannot decode response data from URL %s" % media_url)
                 continue
+            except ExpatError:
```

Comment:
To use this `ExpatError`, you have to import `from xml.parsers.expat import ExpatError` first.